draft-ietf-rats-eat.md

```diff
@@ -323,54 +323,50 @@ An entity may have strong security like defenses against hardware invasive attac
 
-This specification extends those specifications by defining additional claims for attestation.
-This specification also describes the notion of a "profile" that can narrow the definition of an EAT, ensure interoperability and fill in details for specific usage scenarios.
-This specification also adds some considerations for registration of future EAT-related claims.
+EAT uses the name/value pairs the same as CWT and JWT to identify individual claims.
+{{theclaims}} defines common attestation-oriented claims that are added to the CWT and JWT IANA registries.
+As with CWT and JWT, no claims are mandatory and claims not recognized should be ignored.
 
-The identification of a protocol element as an EAT, whether CBOR or JSON encoded, follows the general conventions used by CWT, JWT.
-Largely this depends on the protocol carrying the EAT.
-In some cases it may be by content type (e.g., MIME type).
-In other cases it may be through use of CBOR tags.
-There is no fixed mechanism across all use cases.
+Unlike, but compatible with CWT and JWT, EAT defines claims using Concise Data Definition Language (CDDL) {{RFC8610}}.
+In most cases the same CDDL definition is used for both the CBOR/CWT serialization and the JSON/JWT serialization.
 
-This specification adds one more top-level token type:
+Like CWT and JWT EAT uses COSE and JOSE to provide authenticity, integrity and optionally privacy.
+EAT places new new restrictions on cryptographic algorithms and thus retains all the cryptographic flexibility of CWT, COSE, JWT and JOSE.
+This allows for either symmetric key or public key cryptography and a range of algorithm strengths to accommodate a wide variety of use cases.
 
-* Detached EAT Bundle, {{DEB}}
+EAT goes further than CWT and JWT in allowing nesting of claims sets in several ways to accommodate composite entities that have many subsystems.
+In more closely coupled use cases one claims set is simply nested in another.
+For these the encoding format may not be mixed.
+EAT also allows full tokens with security envelopes to be embedded in an enclosing token.
+This accommodates use cases where the final entity is composed of subordinate entities that produce full EAT tokens on their own.
+In this case the subordinate token may be encoded different than the enclosing token (e.g., a CWT may be enclosed in a JWT).
 
-A detached EAT bundle is structure to hold a collection of detached claims sets and the EAT that separately provides integrity and authenticity protection for them.
-It can be either CBOR or JSON encoded.
+EAT adds the ability to detach claims sets and send them separately from a security enveloped EAT that contains a digest of the detached claims set.
+An example of this is a small and very security attester, perhaps purely built with hardware, attests to claims from outside of it.
+These detached claims sets may be sent with their enveloping EAT together in a message define in this document called a detached EAT bundle.
```

Comment:
```suggestion
These detached claims sets may be sent with their enveloping EAT together in a message defined in this document called a detached EAT bundle.
```

Reply:
I pulled together some similar changes I prepared recently and put then here for comparison/consideration: https://github.com/carl-wallace/eat/commit/22fedf64157750c7f322b3b69467fee84cccf1a6. I had the bullets that appear above in the EAT as a Framework section in the abstract (but in the body may be better). One thing I'd like to see incorporated that I had captured is a reference to the Claims-Set-Claim group in the prose. I'd also prefer the intro to start with some intro words re: EAT vs a pointer to the architecture document.

Reply:
Hi Thomas, you are in agreement with section 1.2 on CWT, JWT... and 1.3 on CDDL, CBOR... and distributing the essential content from them to section 4 intro and the new frameworks section, right?

Part of what's going for me here is I'm trying to respond to comments from the chairs, Eliot and such. They seem to want more explanation. Maybe 5 years from now that will be unnecessary.

I'm not particularly attached to any of this. Will try to find a balance here...

Reply:
hi Laurence, TBH I am fine with anything that has this at its core:

```
> The EAT framework provides:
> * A type system -- the base claims-set & a few aggregation types;
> * Security envelopes based on COSE, JOSE;
> * CBOR and JSON serialisations;
> * A number of pre-defined semantics (the defined "claims") that one
> can readily reuse.
```
which Eliot seemed to be happy with:
```
All good.  In fact, that's so well stated that perhaps you should say it 
in the draft just so.
```

I tend to favour information provided succinctly, but any editorial choices are up to the editors, not to me :-)